### ISSUE TITLE
Wakatime doesn't log run/debug time

### ISSUE THREAD

**Issue by TheYoxy** [wakatime/jetbrains-wakatime#130]
I'm currently using Webstorm 2019.3.3 and it's not logging time when I manually run my project or when I'm attaching the debugger to a node instance.

As of now, I know that this feature has been already included in vscode plugin.

**Comment by alanhamlett** [comment]
After [turning on debug mode](https://github.com/wakatime/jetbrains-wakatime#troubleshooting) are there any error messages in your `idea.log` or `~/.wakatime.log` files?

**Comment by TheYoxy** [comment]
I just get the stacktrace below 3 times in the `idea.log` file: 

```
2020-02-24 10:18:53,606 [  12534]  DEBUG -                       WakaTime -  
com.sun.jna.platform.win32.Win32Exception: The system cannot find the file specified.
	at com.sun.jna.platform.win32.Advapi32Util.registryGetKeys(Advapi32Util.java:1991)
	at com.sun.jna.platform.win32.Advapi32Util.registryGetKeys(Advapi32Util.java:1971)
	at com.wakatime.intellij.plugin.Dependencies.getPythonFromRegistry(Dependencies.java:119)
	at com.wakatime.intellij.plugin.Dependencies.getPythonLocation(Dependencies.java:85)
	at com.wakatime.intellij.plugin.Dependencies.isPythonInstalled(Dependencies.java:45)
	at com.wakatime.intellij.plugin.WakaTime.initComponent(WakaTime.java:88)
	at com.intellij.serviceContainer.MyComponentAdapter.doCreateInstance(MyComponentAdapter.kt:42)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstanceUncached(BaseComponentAdapter.kt:110)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstance(BaseComponentAdapter.kt:72)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstance$default(BaseComponentAdapter.kt:65)
	at com.intellij.serviceContainer.PlatformComponentManagerImpl.createComponents(PlatformComponentManagerImpl.kt:177)
	at com.intellij.openapi.application.impl.ApplicationImpl.lambda$loadComponents$3(ApplicationImpl.java:346)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:163)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:585)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:531)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:59)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:150)
	at com.intellij.openapi.application.impl.ApplicationImpl.loadComponents(ApplicationImpl.java:346)
	at com.intellij.idea.ApplicationLoader$startApp$6$loadComponentInEdtFuture$1.run(ApplicationLoader.kt:204)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1736)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:398)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

**Comment by alanhamlett** [comment]
Can you upgrade to the latest version of the plugin using your IDE settings -> plugins menu, then look for debug messages in your `idea.log` file related to the `wakatime-cli` command?

**Comment by alanhamlett** [comment]
Fixed with v11.0.0.
